javascript/ql/src/semmle/javascript/Collections.qll

```diff
@@ -159,10 +159,9 @@ private module CollectionDataFlow {
    * A step for a `for of` statement on a Map, Set, or Iterator.
    * For Sets and iterators the l-value are the elements of the set/iterator.
    * For Maps the l-value is a tuple containing a key and a value.
-   *
-   * This is partially duplicated behavior with the `for of` step for Arrays (in Arrays.qll).
-   * This duplication is required for the type-tracking steps defined in `CollectionsTypeTracking`.
    */
+  // This is partially duplicated behavior with the `for of` step for Arrays (`ArrayDataFlow::ForOfStep`).
```

Comment:
Hmm, I am actually surprised that this works. I would have thought the line-comment broke the association between the docstring and the class. But let's keep it since it works.